packages/@lwc/integration-tests/README.md

```diff
@@ -7,12 +7,6 @@ End-to-end tests run in the browser across different modes with the option to ru
 
 ## Commands
 
-Builds artifacts in `compat` mode:
-
-```
-yarn build
-```
-
 Builds artifacts in `dev` mode:
 
 ```
```

Comment:
What happens now when you run `yarn build`? The only instruction now is for dev mode with `MODE=dev yarn build`.

Reply:
Looks like it defaults to prod mode. I'll add that in the readme.

Reply:
290d4b569